HW_1/Task2.py

```diff
@@ -0,0 +1,16 @@
+'''
+Задача 2: Найдите сумму цифр трехзначного числа.
+*Пример:*
+123 -> 6 (1 + 2 + 3)
+100 -> 1 (1 + 0 + 0)
+'''
+
+n = int(input('Введите трёхзначное число:'))
+
+if 100 > n:
+    print("Это не трехзначное: ")
+else:
+    a = n // 100
+    b = (n // 10) % 10
```

Comment:
Тут скобки не нужны.
```suggestion
    b = n // 10 % 10
```